### DISCUSSION TITLE
Is it still Alive?

### DISCUSSION THREAD

**Discussion by FabioDiSabatino** [Lyokone/flutterlocation#711] [General]
Hello everyone, I have just started a flutter real-application and I need to fetch user location (at least one time for the moment). 
I have a problem with the **getLocation()** method whenever the GPS is turned-off it stays on loop. 
I found a walkaround but it's not a clear solution. 

I saw there a lot of open issues on this project so I was wondering if @Lyokone is still maintened or if is a dead project.

Thanks.

**Comment by Joren-Thijs-KasparSolutions** [comment]
I to am starting to wonder this since we are looking for Windows Support.

There seems to be a version five in the works but no commits have happend there for a while now it seems?

**Comment by bartekpacia** [comment]
I've been the only person maintaining this plugin since July 2023. If you want to support development of this plugin, either contribute or sponsor me.
